### PR TITLE
docs(config): add stats option `outputPath`

### PR DESCRIPTION
**Pull request by terrierscript**
Append stats option `outputPath`
https://github.com/webpack/webpack/blob/master/lib/Stats.js#L214-L217

### CODE REVIEW

File: src/content/configuration/stats.md [hunk 1]
```diff
@@ -146,6 +146,9 @@ module.exports = {
     // Show dependencies and origin of warnings/errors (since webpack 2.5.0)
     moduleTrace: true,
 
+    // Show outputPath
+    outputPath: true,
```
Comment (EugeneHlushko) [inline src/content/configuration/stats.md]: is this true by default?

Comment (terrierscript) [reply src/content/configuration/stats.md]: @EugeneHlushko 
This default value decided by `forToString`

And I found this document not collected like chunks and source.

* `chunks: true`
  * https://github.com/webpack/webpack/blob/master/lib/Stats.js#L156
* `source: false`
  * https://github.com/webpack/webpack/blob/master/lib/Stats.js#L188

Which case should I suit?

Comment (EugeneHlushko) [reply src/content/configuration/stats.md]: By reading the source code it is not obvious, i would suggest running stats and checking in debugger the default value, can you try that @terrierscript ?

**Comment by terrierscript** [comment]
This PR is stale. I close it

**Comment by EugeneHlushko** [comment]
Hi @terrierscript if we cannot find out default value now, let's change it to `outputPath: bool` and merge. Someone can improve it later

**Comment by terrierscript** [comment]
@EugeneHlushko Sorry, I haven't time to fix this.

File: src/content/configuration/stats.md [hunk 2]
```diff
@@ -146,6 +146,9 @@ module.exports = {
     // Show dependencies and origin of warnings/errors (since webpack 2.5.0)
     moduleTrace: true,
 
+    // Show outputPath
+    outputPath: true | false,
```
Comment (terrierscript) [inline src/content/configuration/stats.md]: @EugeneHlushko Does this mean what you say?

Comment (EugeneHlushko) [reply src/content/configuration/stats.md]: Yes

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**